getting-started.md

```diff
@@ -4,157 +4,94 @@ title: Getting Started with Flutter
-
-## Creating your first sample app
-
-You can use the `flutter` command to create a starter project.
+To create a starter project, open a terminal and run the `flutter` command .
 
 Here is an example:
 
 ```
-$ flutter create -o my_app
+$ flutter create -o myapp
 ```
 
-The above command creates a `my_app` directory that contains a simple demo
+The above command creates a Flutter project directory called `myapp` that contains a simple demo
 app that uses [Material Design](https://www.google.com/design/spec/material-design/introduction.html).
 
-The code for your app is in `my_app/lib/main.dart`.
-To learn more about how to build apps with Flutter, please see the
-[tutorial](/tutorial/).
+In the project directory, the code for your app is in `my_app/lib/main.dart`.
 
-## Setting up your Android device
+## 2. Run your Flutter app
 
-(Don't have an Android device? You can skip down to
-[Setting up your Mac for iOS development](#setting-up-your-mac-for-ios-development).)
+Use the `flutter run` command to run your Flutter app on all connected
+devices and simulators. (The `flutter devices` command will list connected devices and
+simulators.)
 
-Currently Flutter requires an Android device running
-Jelly Bean, v16, 4.1.x or later.
+To run your app from the command-line:
 
- - Install [Android Studio](https://developer.android.com/sdk/) and run through
-   enough of the first-run experience to install the Android SDK. (You shouldn't
-   need to install a JDK.)
+1. Open a terminal and change directories to the root of your app (the same directory that
+contains the `pubspec.yaml` file for your project).
+2. Run the following command.
 
- - Enable developer mode on your device by visiting `Settings > About phone` and
-   tapping the `Build number` field seven times.
+  ```
+  $ flutter run
+  ```
 
- - Enable `Android debugging` in `Settings > Developer options`.
+Alternatively, if you are using the [Atom editor](https://atom.io/) with the
+[Flutter package](https://atom.io/packages/flutter), you can start your Flutter app by selecting
+the `/lib/main.dart` file in the project and pressing the Run App shortcut in the main editor
+screen. The shortcut depends on the operating system of the development machine you are using:
 
- - Using a USB cable, plug your phone into your computer. If prompted on your
-   device, authorize your computer to access your device.
+* **Linux**: `Ctrl+R`
+* **Mac**: `Command+R`
 
- - Check that `adb` lists your device with `adb devices -l`.
 
-By default, Flutter uses the version of the Android SDK where your `adb` tool is based. If
-you want Flutter to use a different installation of the Android SDK, you must set the
-`ANDROID_HOME` environment variable to that specific installation directory.
+If everything works, you should see your starter app on your Android device or iOS Simulator.
 
-## Running a Flutter application on Android
+<div id="starter-app-screenshots">
+  <div class="box2">
```

Comment:
can't the classes go on the `<figure>` elements? I don't really understand why we need the divs at all here.